### PR TITLE
10/reusable hero

### PR DESCRIPTION
**Pull request by NewtonOmbese**
# Description

This pull request is for the Reusable Hero Section issue. This issue is depended upon by other issues hence a reusable component it is. 

To achieve this, we needed to:

```
- Create the component in the articulated path
- Create an exact duplicate of the design provided
- Needed to write a ngIf statement for the footer text
```

## Type of change

- [ ] New feature (non-breaking change which adds functionality)

# Screenshot (optional)
Screenshot from 2023-02-11 03-01-30

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by NewtonOmbese** [comment]
Review and give feedback @IanOdhiambo9

File: tsconfig.base.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
       ],
       "@elewa-group/pages/elewa/home": ["libs/pages/elewa/home/src/index.ts"],
       "@elewa-group/elements/layout": ["libs/elements/layout/src/index.ts"],
-      "@elewa-group/features/components/cards": ["libs/features/components/cards/src/index.ts"],
+      "@elewa-group/features/components/ui-lists": ["libs/features/components/ui-lists/src/index.ts"],
       "@elewa-group/state/data/cms": ["libs/state/data/cms/src/index.ts"],
```
Comment (IanOdhiambo9) [inline tsconfig.base.json:30]: cards component is needed here as well. please retain it too

**Review by IanOdhiambo9 — CHANGES_REQUESTED** [review]
Provide screenshot for mobile view

**Comment by NewtonOmbese** [comment]
Screenshot from 2023-02-13 12-14-57

Mobile view

**Comment by NewtonOmbese** [comment]
Mobile view screenshot provided @IanOdhiambo9

**Comment by NewtonOmbese** [comment]
Review and give feedback @IanOdhiambo9

**Comment by IanOdhiambo9** [comment]
Closing PR due to conflicts and issue status being closed.
Review feedback and apply on the next issue.

**Pull request closed without merging**